app/assets/stylesheets/main.css.scss

```diff
@@ -1925,6 +1925,25 @@ figure.fluidratio {
 }
 
 /*
+  Branding
+*/
+.branding-container {
```

Comment:
One more comment. Should we move these to separate `branding.css.scss`. I feel that the `main` is an unordered trash bin that we shouldn't grow anymore :)